extensions/spark/kyuubi-spark-authz-shaded/pom.xml

```diff
@@ -314,4 +311,27 @@
             </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <!-- Add profile here to exclude jna when enable scala-2.13. -->
+        <profile>
+            <id>scala-2.13</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-shade-plugin</artifactId>
+                        <configuration>
+                            <artifactSet>
+                                <excludes>
```

Comment:
have you tested it? what's the behavior when both `includes` and `excludes` present?

Reply:
It will contain the content in the includes tag and not in the excludes tag.
<img width="1323" alt="image" src="https://github.com/apache/kyuubi/assets/51110188/d69f41ad-d0cb-4d31-9356-e50aff593541">